third_party/go.mod

```diff
@@ -0,0 +1,130 @@
+module github.com/kong/charts/tools
+
+go 1.19
+
+require golang.stackrox.io/kube-linter v0.4.0
```

Comment:
BTW: there's 0.6.0 available https://github.com/stackrox/kube-linter/releases/tag/0.6.0

Reply:
🤔 The module name is `golang.stackrox.io/kube-linter` and the latest version for this is `v0.4.0`. Looks like they missed pushing to the other remote or something?

Reply:
https://github.com/stackrox/kube-linter/issues/494